src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -95,24 +102,26 @@
 
 helps['acs show'] = """
     type: command
-    short-summary: Show details of a container service.
+    short-summary: Show the details for a container service.
 """
 
 helps['acs wait'] = """
     type: command
     short-summary: Wait for a container service to reach a desired state.
+    long-summary: If an operation on a container service was interrupted or was started with `--no-wait`, use this
+                  command to wait for it to complete. Returns
 """
 
 # AKS command help
 
 helps['aks'] = """
      type: group
-     short-summary: Manage Azure Kubernetes clusters.
+     short-summary: Manage Azure Kubernetes Services.
 """
 
 helps['aks browse'] = """
     type: command
-    short-summary: Open a web browser to the dashboard for a managed Kubernetes cluster.
+    short-summary: Show the dashboard for a Kubernetes cluster in a web browser.
     parameters:
         - name: --disable-browser
```

Comment:
Long-term I still think this needs more explanation or an example of why it would be used (since I guess it just sets up an HTTP tunnel?) but that can be a feature request/VSTS task/github issue.

Reply:
I created #5114 to track that.